iree/compiler/Codegen/Passes.cpp

```diff
@@ -62,6 +62,9 @@ LogicalResult verifyLoweringConfiguration(
     case IREE::Codegen::DispatchLoweringPassPipeline::CPUTileFuseAndVectorize:
       return verifyDoubleTilingExpertPassPipelineConfig(op, loweringConfig,
                                                         translationInfo);
+    case IREE::Codegen::DispatchLoweringPassPipeline::LLVMGPUMatmulSimt:
+      return verifyGPUMatmulSimtPassPipeline(op, loweringConfig,
```

Comment:
The verification seems incomplete. More of the configurations need to be verified. I would rather expose it this way after the verification truly verifies the entire configuration.